tests/test_initialisation.py

```diff
@@ -16,6 +16,11 @@ def test_flask_initiates(flask_test_client):
     If this test succeeds then our flask application
     is AT LEAST up and running without errors.
     """
+    flask_test_client.set_cookie(
+        "localhost",
+        "fsd_user_token",
+        "",
+    )
```

Comment:
This was added as we don't want the default cookie used here, added at request of @harryyo